.gitignore

```diff
@@ -0,0 +1 @@
+/parallel-exec/parallel-exec
```

Comment:
Ya I think we should have all `main` packages under a common `cmd` directory too, definitely agree, just sticking with what we have in this repo for now.